extension/options/index.html

```diff
@@ -1,5 +1,5 @@
-<!DOCTYPE html>
-<meta charset="UTF-8">
+<!doctype html>
```

Comment:
It's case-insensitive, but the official one is actually uppercase: https://html.spec.whatwg.org/multipage/syntax.html#the-doctype